### PR TITLE
fixed spacing, imports, and so on to eliminate warnings

### PR DESCRIPTION
**Pull request by Malvolio**

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -2,24 +2,26 @@ import React from "react";
...
-        </Switch>
+      <Switch>
+        <Route path="/p">
+          <PatientApp />
+        </Route>
+        <Route path="/c">
+          <ClinicianApp />
+        </Route>
+      </Switch>
+      <Redirect from="/" to="/p" />
```
Comment (tstanis-snickerbockens) [inline frontend/src/App.tsx:22]: oh good, glad you did that.

File: frontend/src/EncounterList.tsx [hunk 2]
```diff
@@ -1,160 +1,200 @@
...
-        { id: 'time', numeric: false, disablePadding: false, label: 'Scheduled Time' },
-        { id: 'encounter_state', numeric: false, disablePadding: false, label: 'State' },
-        { id: 'patient_connected', numeric: false, disablePadding: false, label: 'Patient Connected' },
-        { id: 'advocate_connected', numeric: false, disablePadding: false, label: 'Advocate Connected' },
-        { id: 'go', numeric: false, disablePadding: false, label: 'Go' },
-      ];
+class EncounterListImpl extends React.Component<
+  EncounterListProps,
+  EncounterListState
+> {
```
Comment (tstanis-snickerbockens) [inline frontend/src/EncounterList.tsx:78]: this belons on the previous line

File: frontend/src/PatientList.tsx [hunk 3]
```diff
@@ -1,142 +1,156 @@
...
-
-    private headers = [
-        { id: 'email', numeric: false, disablePadding: true, label: 'Email' },
-        { id: 'name', numeric: false, disablePadding: false, label: 'Name' },
-        { id: 'advocates', numeric: false, disablePadding: false, label: 'Advocates' }
-      ];
+class PatientListImpl extends React.Component<
+  PatientListProps,
+  PatientListState
+> {
```
Comment (tstanis-snickerbockens) [inline frontend/src/PatientList.tsx:73]: previous line

File: frontend/src/EncounterList.tsx (hunk 2 above)
Comment (Malvolio) [reply frontend/src/EncounterList.tsx:78]: This is done by the automatic formatter.

**Pull request merged**